wg-serving/charter.md

```diff
@@ -0,0 +1,99 @@
+# WG Serving Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance serving workloads on Kubernetes, specifically focused on
+hardware-accelerated AI/ML inference.
+
+The working group will focus on the novel challenges of compute-intensive online
+inference. Scenarios solving use cases involving non-fungible accelerators will
+be prioritized over solution against generic CPU. However all improvements
+should, where possible, benefits other serving workloads like web services or
+stateful databases, be usable as primitives by multiple ecosystem projects, and
+compose well into the workflows of those deploying models to production.
+
+The Working Group Batch has similar scope. The difference in scope by simplified
+definition, is that the Serving WG  will generally concentrate on the workloads
+where Pods are running with `restartPolicy=Always`, while WG Batch will
+generally be looking at Pods with the `restartpolicy=OnFailure`. There are edge
+cases to this definition, but it creates an easy enough framework to
+differentiate the scope these two Working Groups.
```

Comment:
I really like this improvement. The section on the job types makes it quite clear. 👍

Reply:
thanks!